module-4/backend/week_two.md

```diff
@@ -5,18 +5,56 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
+
+  ES5 uses Var and ES6 uses const and let
+  
 2. What's the difference between asynchronous and synchronous JavaScript? 
+
+  Synchrounous means that only one block of code can run at a single time. Asyncrounous means that we can use calls like AJAX   to run while the page loads so that the page doesn't hang on loading.
+  
 3. What are the four pillars of Object Oriented programming?
+
+  Polymorphism, Abstraction, Encapsulation, Inheritance
+  
 4. What are some tools available in JavaScript to help you write object oriented code?
+
+  Using classes or prototypes and POJOs
+  
+ 
 5. What are some key concepts to be aware of when refactoring your JavaScript?
+
+  Break functions and parts of code up into separate files i.e eventHandler, request, handler
+  
 6. What's a callback function and what are some reasons when we use/need callback functions?
+
+  A call back function is when you call a function in a parameter of another function. You do this to execute a function after another function is completed.
+  
 7. What's the scope of variables in Javascript?
+
+  Global and local
+  
 8. What's the difference between `==` and `===` in JavaScript?
-9. Why do front end frameworks exist?
+
+  '==' evaluates abstact equality and '===' evalutes a strict equality
+  
+9.  Why do front end frameworks exist?
+
+  Frontend frameworks exist as a way to construct a design for an app. Things have to be done a certain way in frameworks to force the design of the app.
```

Comment:
maybe not necessarily "design", but the structure and organization of the application